lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -638,8 +639,9 @@ class MuteLayer : public BaseCmd
             saveSelection();
             stage->MuteLayer(layer->GetIdentifier());
         }
+
         // we can release the pointer
-        _mutedLayer = nullptr;
+        removeMutedLayer(layer);
```

Comment:
Question here: In AddMutedLayer, we recursively added all dirty sublayers to the mutedLayer list, so when remvoeMutedLayer should we do the same? For now it just removes one layer.

Reply:
You're right, I will look into it.